pkg/codegen/extension.go

```diff
@@ -6,59 +6,55 @@ import (
 )
 
 const (
-	extPropGoType    = "x-go-type"
-	extPropOmitEmpty = "x-omitempty"
-	extPropExtraTags = "x-go-extra-tags"
-	extMiddlewares   = "x-go-middlewares"
+	extPropGoType        = "x-go-type"
+	extPropOmitEmpty     = "x-omitempty"
+	extPropExtraTags     = "x-go-extra-tags"
+	extPropOptionalValue = "x-go-optional-value"
+	extPropString        = "x-go-string"
+	extMiddlewares       = "x-go-middlewares"
 )
 
```

Comment:
Most of these changes are just trivial abstracting-things-inside-a-function. They're not very important changes.